specs/operator/addon.spec

```diff
@@ -78,4 +78,40 @@ Importance: Critical
 Pos/Neg: Negative
 
 Steps:
-  * Update addon config with clusterTasks as "false" communityClustertasks as "false" and pipelineTemplates as "true" and expect message "validation failed: pipelineTemplates cannot be true if clusterTask is false"
\ No newline at end of file
+  * Update addon config with clusterTasks as "false" communityClustertasks as "false" and pipelineTemplates as "true" and expect message "validation failed: pipelineTemplates cannot be true if clusterTask is false"
+
+## Disable/Enable resolverTasks: PIPELINES-15-TC06
+Tags: e2e, integration, resolverTasks, admin, addon, sanity
+Component: Pipelines
+Level: Integration
+Type: Functional
+Importance: Critical
+
+Steps:
+  * Update addon config with resolverTasks as "false" and expect message ""
+  * Tasks "s2i-java" are "not present" in namespace "openshift-pipelines"
+  * Update addon config with resolverTasks as "true" and expect message ""
+  * Tasks "s2i-java" are "present" in namespace "openshift-pipelines"
+
+## Disable/Enable resolverTasks with additional Tasks: PIPELINES-15-TC07
+Tags: e2e, integration, resolverTasks, admin, addon, sanity
```

Comment:
Please keep `sanity` tag only on one of these two scenarios. This is what we run when operator build is in stage and we want to the test suite to execute as quickly as possible.

Reply:
got it now. changed to have sanity only for the first test.